methods/transient/Transient_base.ipynb

```diff
@@ -0,0 +1,1390 @@
+{
```

Comment:
<p>Need some description of what this file is saving.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>